client/src/__tests__/HomePage.test.jsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import HomePage from '../components/HomePage';
+
+test('HomePage renders child components', () => {
+  render(
+    <Router>
+        <HomePage />
```

Comment:
This is more of a general comment. Do you need to pass down props to this component to get it to work? Think about it!